docs/modules/components/pages/processors/ollama_embeddings.adoc

```diff
@@ -0,0 +1,113 @@
+= ollama_embeddings
+:type: processor
+:status: experimental
+:categories: ["AI"]
+
+
+
+////
+     THIS FILE IS AUTOGENERATED!
+
+     To make changes, edit the corresponding source file under:
+
+     https://github.com/redpanda-data/connect/tree/main/internal/impl/<provider>.
+
+     And:
+
+     https://github.com/redpanda-data/connect/tree/main/cmd/tools/docs_gen/templates/plugin.adoc.tmpl
+////
+
+
+component_type_dropdown::[]
+
+
+Processor that uses the Ollama API to create vector embeddings.
+
+Introduced in version 4.32.0.
+
+
+[tabs]
+======
+Common::
++
+--
+
+```yml
+# Common config fields, showing default values
+label: ""
+ollama_embeddings:
+  server_address: "" # No default (optional)
+  model: nomic-embed-text # No default (required)
+  text: "" # No default (optional)
+```
+
+--
+Advanced::
++
+--
+
+```yml
+# All config fields, showing default values
+label: ""
+ollama_embeddings:
+  server_address: "" # No default (optional)
+  ollama_directory: "" # No default (optional)
+  model: nomic-embed-text # No default (required)
+  text: "" # No default (optional)
+```
+
+--
+======
+
+Processor that uses the Ollama API to create vector embeddings.
+
+See https://ollama.com/[https://ollama.com/^] for more information.
+
+== Fields
+
+=== `server_address`
+
+The address of a remote ollama server to use. If omitted a local ollama server will be started and ran.
+
+
+*Type*: `string`
+
+
+=== `ollama_directory`
+
+The directory to unpack the ollama binary into. Defaults to /tmp
+
+
+*Type*: `string`
+
+
+=== `model`
+
+The ollama model to use. A full list is available at ollama.com/models
+
+
+*Type*: `string`
+
+
+```yml
+# Examples
+
+model: nomic-embed-text
+
+model: mxbai-embed-large
+
+model: snowflake-artic-embed
+
+model: all-minilm
+```
+
+=== `text`
+
+The text to create embeddings for. By default the entire payload as a string is submitted.
```

Comment:
The text you want to create vector embeddings for. By default, the processor submits the entire payload as a string.